python/compiler/aiecc/configure.py.in

```diff
@@ -17,6 +17,21 @@ aie_unified_compile = True
 host_disable_compile = not pythonize_bool("@AIECC_HOST_COMPILE@")
 host_architecture = "@LLVM_HOST_TRIPLE@"
 
+peano_install_dir = "@PEANO_INSTALL_DIR@"
+if peano_install_dir == "":
+    peano_install_dir = "peano_not_found"
```

Comment:
These lines seem redundant?

Reply:
I guess I wasn't sure what `os.path.exists("")` would return. Double checking now it does return `false` so redundant is correct!